.ci/install_runtime.sh

```diff
@@ -49,7 +49,7 @@ PKGDEFAULTSDIR="${SHAREDIR}/defaults/kata-containers"
 NEW_RUNTIME_CONFIG="${PKGDEFAULTSDIR}/configuration.toml"
 # Note: This will also install the config file.
 build_and_install "${runtime_repo}" "" "true" "${tag}"
-experimental_qemu="${experimental_qemu:-false}"
+shared_fs="${shared_fs:-9pfs}"
```

Comment:
I think this ability to switch between 9p and virtiofs would be useful to 2.x too. For example, I just set a commit (https://github.com/kata-containers/tests/pull/3175/commits/0f1b6d915774ba5b0dec128855e7bef796bff4c0) that changes the shared_fs when running on openshift-ci. What you proposing here is much better.

Reply:
@wainersm I see , I think we could have something like this on 2.0 that allows to propagate the shared backend to the kata-runtime config. So we can explicitly what the openshift  should be testing.